src/device_hub.h

```diff
@@ -18,10 +18,21 @@ namespace librealsense
         explicit device_hub(std::shared_ptr<librealsense::context> ctx, int vid = 0, bool register_device_notifications = true);
 
         /**
-         * If any device is connected return it, otherwise wait until next RealSense device connects.
-         * Calling this method multiple times will cycle through connected devices
+         * The function implements both blocking and non-blocking device generation functionality based on the input parameters:
+         * Calling the function with zero timeout results in searching and fetching the device specified by the `serial` parameter
+         * from a list of connected devices. The call will return nullptr if none is found.
+         * Calling the function with a valid timeout will block till the resulted device is found or the timeout expires.
+         *
+         * \param[in] timeout_ms  The waiting period for the requested device to be reconnected (milliseconds).
+         * Set it to zero to avoid blocking the thread execution
+         *\param[in] loop_through_devices  - promote internal index to the next available device that will be retrieved on successive call
+         * \param[in] serial  The serial number of the requested device. Use empty pattern ("") to request for "any suitable" device.
```

Comment:
what do you mean "suitable"? any connected...

Reply:
"RealSense" as opposed to a platform cam. I'll correct this